Dockerfile

```diff
@@ -26,7 +21,10 @@ ARG CISA_UID=421
 ARG CISA_GID=${CISA_UID}
 ARG CISA_USER="cisa"
 ENV CISA_GROUP=${CISA_USER}
-ENV CISA_HOME="/home/${CISA_USER}"
+ENV CISA_HOME="/home/${CISA_USER}" \
+    ADMIRAL_CONFIG_FILE="/run/secrets/admiral.yml" \
+    ADMIRAL_CONFIG_SECTION="dev-mode" \
+    ADMIRAL_WORKER_NAME="dev"
```

Comment:
Admiral-related environment variables should be moved to their own section and specified one per line:
```suggestion
ENV CISA_HOME="/home/${CISA_USER}"

###
# Admiral configuration variables
###
ENV ADMIRAL_CONFIG_FILE="/run/secrets/admiral.yml"
ENV ADMIRAL_CONFIG_SECTION="dev-mode"
ENV ADMIRAL_WORKER_NAME="dev"
```